### PR TITLE
More formatting and style for publikationer-page

### PR DESCRIPTION
**Pull request by stenwall**

### CODE REVIEW

File: css/scss/_lists.scss [hunk 1]
```diff
@@ -1,32 +1,44 @@
...
+        transition: transform 0.35s ease-in-out;
+      }
+    }
+  }
+}
+
+#white-list {
+  li {
+    a {
+      color: white;
```
Comment (axelra82) [inline css/scss/_lists.scss]: Missing $ for variable:
```suggestion
      color: $white;
```

File: css/scss/pages/_publikationer.scss [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+	row-gap: 2rem;
+	grid-template-areas:
+	  ". ."
+	  "full-width full-width";
+  }
+
+  	#article-section {
+		section {
+			.pub-head {
+				grid-area: "header";
```
Comment (axelra82) [inline css/scss/pages/_publikationer.scss]: Should be:
```suggestion
				grid-area: header;
```
without the quotes 😉

**Review by axelra82 — CHANGES_REQUESTED** [review]
Aside from line comments all looks goods. I would however recommend re-thinking the 4 col repeat "inside a box, inside a box" on the article grid. On a medium-sized screen this might be just enough to make the columns go outside the grid (overflow).

Maybe 3 cols are more suitable for the amount of content in them? 🤔

Also, since the title in each article in the grid will have unknown lengths, maybe it would be more aesthetically pleasing to start with an image (with the "download/more info" option bellow), then publish date, then title. This way they would all "line up" 😊

**Comment by stenwall** [comment]
I agree! A lot of the things your are pointing ut are not really done yet, and of course I'm going to fix all the typos! I was really tired there yesterday... Good idea about the columns, I'm going to look at that. About the white list, is there a smarter way of doing it?

**Comment by axelra82** [comment]
> ... About the white list, is there a smarter way of doing it?

If you're thinking of the `::before{content:'...';}` part then yes, there is an alternative solution 🤓

We could **break out the SVG data**, to **separate SVG files** (e.g. _/assets/svg/icon-chevron-right.svg_) and then use the css **mask** property. This was not a viable option many years ago when it first came (because of lacking wider browser support), as such it didn't occur to me that we could use it. [The browser support](https://caniuse.com/css-masks) is close to 93% now, and works in latest Safari, Firefox, Chrome and Edge (not I.E 11).

**Here are two links regarding this issue:**
[CSS-Tricks](https://css-tricks.com/almanac/properties/m/mask-image/)
[SO question](https://stackoverflow.com/questions/21509982/change-svg-fill-color-in-before-or-after-css)

[**and MDN docs**](https://developer.mozilla.org/en-US/docs/Web/CSS/mask)

**NOTE:** In this case it would work just fine (_although I'm not 100% sure there is anything to be gained since it's such a small project. But for good measure I think it's nice to show that there are alternativs_ 😊). **However, keep in mind that** this (see below) solution will apply one color to the entire graphic. If we had more complex vectors where we wanted different colors for different parts, it wouldn't work.

**Change current li to:**
```
li{
  a{
    &::before{
      content: "";
      mask: url("/assets/svg/icon-chevron-right.svg") center center no-repeat;
      -webkit-mask: url("/assets/svg/icon-chevron-right.svg") center center no-repeat;
      background: $black;
    }
  }
}
```
**and then use:**
```
.white-list{
  li{
    a{
      &::before{
        background: $white;
      }
    }
  }
}
```
**to change the graphic color.**

**Comment by stenwall** [comment]
> **Change current li to:**
> 
> ```
> li{
>   a{
>     &::before{
>       content: "";
>       mask: url("/assets/svg/icon-chevron-right.svg") center center no-repeat;
>       -webkit-mask: url("/assets/svg/icon-chevron-right.svg") center center no-repeat;
>       background: $black;
>     }
>   }
> }
> ```
^ Is this supposed to go in the `_lists.scss`? Under `ul` like you wrote or under `#white-list`? If under `ul`, should I just remove `#white-list`?

> ```
> .white-list{
>   li{
>     a{
>       &::before{
>         background: $white;
>       }
>     }
>   }
> }
> ```
^ Does this go in my "local" list then? 

Sorry for all the stupid questions! At least I'm learning a lot 😅

**Comment by axelra82** [comment]
Suggestion for mixing (which was discussed in greater depth durring daily slack group call).

```
@mixin chevron-right($color: $black) {
	content: url('data:image/svg+xml;utf8,<svg aria-hidden="true" focusable="false" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512"><path fill="#{sansHashColor($color)}" d="M285.476 272.971L91.132 467.314c-9.373 9.373-24.569 9.373-33.941 0l-22.667-22.667c-9.357-9.357-9.375-24.522-.04-33.901L188.505 256 34.484 101.255c-9.335-9.379-9.317-24.544.04-33.901l22.667-22.667c9.373-9.373 24.569-9.373 33.941 0L285.475 239.03c9.373 9.372 9.373 24.568.001 33.941z"></path></svg>');
}
@include chevron-right(red);
```

**Comment by stenwall** [comment]
changes done! @axelra82

**Review by axelra82 — APPROVED** [review]

**Comment by axelra82** [comment]
@stenwall looks like everything is 👌 now.

Just the remaining new style.min.css conflict, which I'll leave for @Oskar-Mikael to handle 🤓

**HINT:** deleting line 1 and 3-5 would resolve the conflict by accepting the incoming change (and thereby discarding the existing code) 😉

**Comment by Oskar-Mikael** [comment]
Remaining conflict resolved. Good to go with merge.

**Review by Oskar-Mikael — COMMENTED** [review]
Remaining conflict resolved. Good to go with merge.

**Pull request merged**